### DISCUSSION TITLE
Repository not opening in Codespaces in VS Code

### DISCUSSION THREAD

**Discussion by RebeccaCrecelius** [community/community#108303] [Codespaces]
### Select Topic Area

Bug

### Body

One of my repositories is not opening in VS Code. A previous one was still listed. When the one I wanted to connect did not connect, I tried another. The third connected. The repository is seen by VS Code, and I am asked to select a branch. However, after one is selected, nothing happens. It does not connect.

**Comment by SuperZekes** [comment]
Delete all your codespaces and try to make one again thats all I could say.

**Reply by RebeccaCrecelius** [reply]
The issue was too many workspaces.

**Reply by SuperZekes** [reply]
Okay so I may have been right and wrong in a way.
